frontend/model/state.js

```diff
@@ -22,6 +22,7 @@ import * as EVENTS from '~/frontend/utils/events.js'
 import './contracts/group.js'
 import './contracts/mailbox.js'
 import './contracts/identity.js'
+import { captureLogsStart, captureLogsPause } from '~/frontend/model/captureLogs.js'
```

Comment:
Make sure to edit the `initialState` object to add the default enabled captured filters (`['info', 'warn', 'error']`)

Reply:
On Figma it says the default is warn and error and info wasn't included. You mentioned debug shouldn't be included by default because it may be heavy to store so many "debug" logs. Do you really want to add info (or debug?) or should we keep as it is? (only warn and error)

Reply:
We almost never use `info`. So having it be `info`, `warn`, and `error` should be fine. We can change the default in the future if needed.